### PR TITLE
Migrate the remaining specs and remove protractor related files

### PR DESCRIPTION
**Pull request by RFSH**
This PR migrates the last two remaining specs:
-  `immutable-inputs.spec.js` is now `immutable-spec.ts`.
- `submission-disabled.spec.js` content was moved to `errors.spec.ts`.

Apart from removing protractor related files, I also added [test retries](https://playwright.dev/docs/test-retries) to the histogram spec given how flaky it is.

### CODE REVIEW

File: test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    await test.step('submit and save the data', async () => {
+      await testSubmission(page, testParams.create.submission);
+    });
+  });
+
+  test('Edit mode', async ({ page, baseURL }, testInfo) => {
+    await test.step('open recordedit page', async () => {
+      await page.goto(`${baseURL}/recordedit/#${getCatalogID(testInfo.project.name)}/${testParams.schemaTable}/id=2`);
+      await RecordeditLocators.waitForRecordeditPageReady(page);
+      await page.pause();
```
Comment (jrchudy) [inline test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts:193]: left over `page.pause()` to be removed

File: test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                const props = RecordeditLocators.getTimestampInputsForAColumn(page, col.name, 1);
+                await testAutomaticallyGenerated(props.time, col.displayname);
+                await testAutomaticallyGenerated(props.date, col.displayname);
+                break;
+              case RecordeditInputType.FILE:
+              case RecordeditInputType.FK_DROPDOWN:
+              case RecordeditInputType.FK_POPUP:
+              case RecordeditInputType.BOOLEAN:
+                await expect.soft(RecordeditLocators.getInputPlaceholderMessage(page, col.name, 1)).toHaveText('Automatically generated');
+                // TODO
```
Comment (jrchudy) [inline test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts:173]: Was this TODO meant for something else?

**Review by jrchudy — APPROVED** [review]
Changes look good. My inline comments are for cleaning up the code further.

File: test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts (hunk 2 above)
Comment (RFSH) [reply test/e2e/specs/default-config/recordedit/immutable-inputs.spec.ts:173]: I added this before adding the `getInputPlaceholderMessage` locator and forgot to remove it once it was implemented.

**Pull request merged**